question28/js/password-done.js

```diff
@@ -0,0 +1,4 @@
+const url = new URL(window.location.href);
+const params = url.searchParams;
+
+if(params.get('token') !== localStorage.getItem('takenForSetNewPassword')) window.location.href = '../notautherize.html';
```

Comment:
②Obtain the parameters, compare them with the token in the local storage, and if they do not match, transition to notautherize.html.